userspace/libsinsp/filter.cpp

```diff
@@ -2040,30 +2044,64 @@ void sinsp_evttype_filter::add(string &name,
 			filters->push_back(wrap);
 		}
 	}
+
+	for(auto &tag: tags)
+	{
+		auto it = m_filter_by_tag.find(tag);
```

Comment:
Use lower_bound() instead of find() and pass the iterator as a hint to emplace. It requires a little extra checking but saves a map walk during emplace.